cmake/Modules/FindTPLCUDA.cmake

```diff
@@ -11,9 +11,16 @@ if(KOKKOS_CXX_HOST_COMPILER_ID STREQUAL NVHPC AND CMAKE_VERSION VERSION_LESS "3.
   message(FATAL_ERROR "Using NVHPC as host compiler requires at least CMake 3.20.1")
 endif()
 
+set(TPL_CUDA_LIBRARIES "")
+if(KOKKOS_ENABLE_COMPILE_AS_CMAKE_LANGUAGE)
+  set(TPL_CUDA_LIBRARIES CUDA::cuda_driver)
+else()
+  set(TPL_CUDA_LIBRARIES CUDA::cuda_driver CUDA::cudart)
+endif()
+
```

Comment:
could you add a FIXME and a short description or a link to the issue? We might want to fix this for the non-language build eventually (e.g. Kokkos 5 ... )